code/game/objects/items/storage/pouch.dm

```diff
@@ -538,6 +538,45 @@
 	new /obj/item/explosive/grenade/incendiary/som(src)
 	new /obj/item/explosive/grenade/incendiary/som(src)
 
+/obj/item/storage/pouch/mine
+	name = "mine pouch"
+	desc = "To keep your death traps tucked safely."
+	icon_state = "large_explosive"
+
+/obj/item/storage/pouch/mine/Initialize(mapload, ...)
+	. = ..()
+	storage_datum.sprite_slots = 3
+	storage_datum.storage_slots = 5
+	storage_datum.max_storage_space = 10
+	storage_datum.set_holdable(can_hold_list = list(/obj/item/mine))
+
+/obj/item/storage/pouch/mine/flash
+	name = "flash mine pouch"
+	fill_type = /obj/item/mine/flash/battery_included
+	fill_number = 5
+
+/obj/item/storage/pouch/mine/flash/Initialize(mapload)
+	. = ..()
+	//Extra 2 space and 1 slot for the spare battery
+	storage_datum.max_storage_space = 12
+	storage_datum.storage_slots = 6
```

Comment:
subtype of a pouch just having extra capacity 4noreason (when it can be refilled) is ??